kong.conf.default

```diff
@@ -1334,3 +1334,19 @@
                                  # server.
                                  #
                                  # See https://github.com/openresty/lua-nginx-module#lua_socket_pool_size
+
+#untrusted_lua = off             # off | sandbox | on
```

Comment:
I thought we wanted to disable custom lua completely by default, but I misread the intent.

Apparently the right default is `untrusted_lua=sandbox`.

My bad.